shell/app/modules/cmp/pages/alarm-report/report-records/index.tsx

```diff
@@ -167,13 +167,15 @@ export default () => {
                   </li>
                 ))}
               </ul>
-              <Pagination
-                className="text-center mt12"
-                simple
-                defaultCurrent={1}
-                total={total}
-                onChange={(no) => handleChange(no)}
-              />
+              {total && (
```

Comment:
I dont quite understand how could it be page count is 3 but current page is 0( in your snapshot).
if total is 0 then page count should be 0 as well. 
should this pagination add a current prop?

Reply:
In the first render, the total is 0, whatever our defaultCurrent set will be reset to 0;
When we get the total from the API, the total will be updated, but defaultCurrent won't be updated.
I don't think we should add current props here, what we want here is just a defaultCurrent, just leave the current control by Pagination component